card/mod/standard/set/all/error.rb

```diff
@@ -9,222 +9,45 @@ def copy_errors card
     ""
   end
 
-  view :unsupported_view, perms: :none, tags: :unknown_ok, error_code: 404 do
-    tr(:unsupported_view, view: voo.unsupported_view, cardname: error_cardname)
-  end
-
   view :missing, perms: :none do
     ""
   end
 
-  view :not_found, perms: :none, error_code: 404 do
-    error_name = card.name.present? ? safe_name : tr(:not_found_no_name)
-    tr(:not_found_named, cardname: error_name)
-  end
-
-  view :server_error, perms: :none, error_code: 500 do
+  view :server_error, perms: :none do
     tr(:server_error)
   end
 
-  view :denial, perms: :none, error_code: 403 do
+  view :denial, perms: :none do
     focal? ? tr(:denial) : ""
   end
 
-  view :bad_address, perms: :none, error_code: 404 do
-    tr(:bad_address)
+  view :not_found, perms: :none do
+    error_name = card.name.present? ? safe_name : tr(:not_found_no_name)
+    tr(:not_found_named, cardname: error_name)
   end
 
-  view :too_deep, perms: :none, closed: true do
-    tr(:too_deep)
+  view :bad_address, perms: :none do
+    raise Card::Error::OpenError, tr(:bad_address)
```

Comment:
Besides the name "OpenError" I don't like this approach conceptually. The OpenError class is kind of a contradiction in itself. An exception that is so common that we show it to the user can't be exceptional. Like this bad address here. It's not an exception. It's a very common user input. If we raise an exception then Ruby puts a lot of effort in generating debug info (like a stack trace) that we never use because there is no bug to find. The more efficient way to resurface from the deep nested block sea is to use `throw` and `catch`.